nixos/modules/system/boot/unl0kr.nix

```
@@ -0,0 +1,91 @@
+{ config, lib, pkgs, utils, ... }:
```

```suggestion
{ config, lib, pkgs, ... }:
```

`utils` is not used here.